src/commands/addons/admin/manifest/pull.ts

```diff
@@ -73,11 +73,10 @@ export default class Pull extends CommandExtension {
     .catch((err:any) => {
       if (err) {
         if (slug) {
-          this.error(`Unable to make get data on a slug with the name of ${color.blue(slug)}`)
+          this.error(`Unable to fetch manifest for add-on ${color.blue(slug)}`)
         } else {
-          this.error('Please make sure you have a slug.')
+          this.error('Please make sure you have a provided a slug to fetch.')
```

Comment:
@mathias should this be 'Please make sure you have provided a slug to fetch.'?

Reply:
Fixed in 1e5632e268abe7f786378c303a6447e0f661f50d